### PR TITLE
bump build nr

### LINKED ISSUE #47: python 3.11 version
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

No 3.11 version built.

### Installed packages

```shell
none
```

### Environment info

```shell
none
```

### PR DESCRIPTION
**Pull request by marscher**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Fixes #47

### CODE REVIEW

**Comment by marscher** [comment]
@conda-forge-admin please rerender

**Comment by mbargull** [comment]
Your can cherry-pick this commit to add Py3.11: https://github.com/conda-forge/jpype1-feedstock/pull/45/commits/779602410eb4e7e213b1f0893a57f2bb82cfedfc
(and rerender again afterwards)

**Comment by marscher** [comment]
@conda-forge-admin please rerender

File: conda-forge.yml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 provider:
   linux_aarch64: default
   linux_ppc64le: default
   win: azure
 test_on_native_only: true
-github:
-  branch_name: main
-  tooling_branch_name: main
-conda_build:
-  pkg_format: '2'
```
Comment (mbargull) [inline conda-forge.yml:13]: The last merge removed this. Can you please add it back?

**Review by mbargull — CHANGES_REQUESTED** [review]

File: conda-forge.yml (hunk 1 above)
Comment (marscher) [reply conda-forge.yml]: Thanks, good catch!

**Comment by mbargull** [comment]
@conda-forge-admin, please rerender

**Pull request merged**